### ISSUE TITLE
Calculation after an event finished

### ISSUE THREAD

**Issue by Corzoo** [Js41637/Overwatch-Item-Tracker#300]
Wouldn't it be better to lower the price for each item to the price of next years event after an event finished?
For example Year of the Pig event, the new Hanzo Legendary 'Huang Zhong' is still in the list for 3000 Coins while it will be 1000 the next time you will be able to actually pay for the skin.

If the Item Tracker would change those prizes in advance, players would be able to better plan their purchases with the actual amount of Coins they will need. Right now the 'Cost to purchase remaining items' prize is always higher than the actual cost you can expect. (Thats an example, all the sub pages have also higher total prices because of that)

**Comment by Js41637** [comment]
Yeah I've wanted to do this but currently the only way it knows to increase the price is if an item is new and I don't want to remove that between events. I don't store the individual price on every item because the price is generally predictable.

I need to add some logic to check if an event has ended and if it has, don't increase the price.

**Comment by GingerRoot** [comment]
When the Anniversary event comes, you're able to buy the items from other events. However, any "new" items are still sold at their increased price.
The prices don't drop until the next time that event comes out.

**Comment by Js41637** [comment]
GinerRoot does have a good point. I thought there was a reason I was against the idea. I guess after anniversary all the prices can be lowered but as I mentioned above. I don't really have a method of doing this yet.
